quantities/src/main/java/com/decathlon/vitamin/compose/quantities/VitaminQuantitiesColors.kt

```diff
@@ -0,0 +1,24 @@
+package com.decathlon.vitamin.compose.quantities
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+class QuantityColors(
+    val textColor: Color,
+    val borderColor: Color,
+    val backgroundColor: Color
+)
+
+object VitaminQuantitiesColors {
+    @Composable
+    fun default(
+        textColor: Color = VitaminTheme.colors.vtmnContentPrimary,
+        borderColor: Color = VitaminTheme.colors.vtmnBorderInactive,
+        backgroundColor: Color = VitaminTheme.colors.vtmnBackgroundPrimary
+    ): QuantityColors = QuantityColors(
```

Comment:
Please use `remember` to avoid performance issue:

```kotlin
object VitaminQuantitiesColors {
    @Composable
    fun default(
        textColor: Color = VitaminTheme.colors.vtmnContentPrimary,
        borderColor: Color = VitaminTheme.colors.vtmnBorderInactive,
        backgroundColor: Color = VitaminTheme.colors.vtmnBackgroundPrimary
    ): QuantityColors = remember(textColor, borderColor, backgroundColor) {
        QuantityColors(
            borderColor = borderColor,
            backgroundColor = backgroundColor,
            textColor = textColor
        )
    }
}
```

Check other components if you need more information about that.